sklearn/linear_model/ridge.py

```diff
@@ -111,7 +111,7 @@ def _solve_cholesky(X, y, alpha):
         return linalg.solve(A, Xy, sym_pos=True,
```

Comment:
what happens in this case? Does `linalg.solve` returns the correct dtype? is it tested?